ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -118,6 +120,44 @@ const EthOverview = ({ className, showAddress }) => {
   }
 
   const isSwapsChain = useSelector(getIsSwapsChain);
+  const isSigningEnabled = useSelector((state) =>
```

Comment:
Seems like we need to check if the account can perform sendTransaction and sendUserOperation